### PR TITLE
PR: Create new demo file

### PR DESCRIPTION
**Pull request by pcandoalmeida**

### CODE REVIEW

**Comment by IceWindFour** [comment]
Looks good to me

File: demo.py [hunk 1]
```diff
@@ -0,0 +1 @@
+This is a sample file
```
Comment (pcandoalmeida) [inline demo.py:1]: This looks good

Comment (pcandoalmeida) [reply demo.py:1]:
```suggestion
This is a sample file
```

**Review by pcandoalmeida — COMMENTED** [review]
LGTM

**Pull request merged**